src/EventListener/AssetsInstallCommandListener.php

```diff
@@ -75,7 +73,7 @@ public function copySonataCoreBundleAssets(ConsoleTerminateEvent $event)
         $this->execute($event->getInput(), $event->getOutput(), $application);
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
+    private function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
```

Comment:
This is a BC break.

And the parent method execute is protected.

Reply:
Oh it's final

Reply:
I thought the same first 😄